lib/bgs/service.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module BGS
+  class Service
+    def initialize(user)
+      @user = user
+    end
+
+    def create_relationship(relationship_params)
+      service.vnp_ptcpnt_rlnshp.vnp_ptcpnt_rlnshp_create(relationship_params.merge(bgs_auth))
+    end
+
+    def bgs_auth
+      {
+        jrn_dt: Time.current.iso8601,
+        jrn_lctn_id: Settings.bgs.client_station_id,
+        jrn_status_type_cd: 'U',
```

Comment:
Mind creating a constant for this? It might better describe what it is for someone like me who has no context. e.g.

```ruby
FOO_BAR_STATUS_TYPE = 'U'.freeze
# ....
jrn_status_type_cd: FOO_BAR_STATUS_TYPE
```

Reply:
Good idea, thanks - done.

Reply:
Rubocop yelled at me for having a ".freeze", so I removed that part.